extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -483,12 +483,15 @@ export class AmpA4A extends AMP.BaseElement {
     }
     // If in localDev `type=fake` Ad specifies `force3p`, it will be forced
     // to go via 3p.
-    if (getMode().localDev &&
-        this.element.getAttribute('type') == 'fake' &&
-        this.element.getAttribute('force3p') == 'true') {
-      this.adUrl_ = this.getAdUrl();
-      this.adPromise_ = Promise.resolve();
-      return;
+    if (getMode().localDev && this.element.getAttribute('type') == 'fake') {
+      // Prevent issuing a second network request to try to fetch
+      // the same JSON again.
+      this.experimentalNonAmpCreativeRenderMethod_ = XORIGIN_MODE.NAMEFRAME;
```

Comment:
Ah, I see what you mean. This is due to the setting of "method" here: https://github.com/ampproject/amphtml/blob/49951ce9047a634fdcfc04e49fa1ca3dcff92e9a/extensions/amp-a4a/0.1/amp-a4a.js#L611 
In prod, the HTTP header will be set. But when running locally, it isn't. Without explicitly setting it here, we hit the error on line 615 when testing locally.